genrl/__init__.py

```diff
@@ -1,14 +1,12 @@
-from genrl.deep import DDPG, DQN, PPO1, SAC, TD3, VPG, A2C
-from genrl.classical import (
-    Bandit,
-    GaussianBandit,
-    BernoulliBandit,
-    BanditPolicy,
-    EpsGreedyPolicy,
-    UCBPolicy,
-    SoftmaxActionSelectionPolicy,
-    BayesianUCBPolicy,
-    ThompsonSamplingPolicy,
-    SARSA,
-    QLearning,
-)
+from genrl.classical import SARSA  # noqa; noqa; noqa
```

Comment:
Why are there 3 separate `noqa`s? And is it actually necessary to separate imports like this?

Reply:
Lol, I have no clue where this came from. I just added 1